skyportal/handlers/api/gcn.py

```diff
@@ -118,14 +122,10 @@ def post(self):
             DBSession().add(localization)
             DBSession().commit()
 
-            contour_func = functools.partial(
-                add_contour,
-                localization.id,
-                self,
-            )
-            IOLoop.current().run_in_executor(None, contour_func)
+            log(f"Generating tiles/contours for localization {localization.id}")
 
-        self.verify_and_commit()
+            IOLoop.current().run_in_executor(None, lambda: add_tiles(localization.id))
```

Comment:
Question: is there no permission checking surrounding adding of tiles and contours?

Reply:
I am not sure why we would have ownership of those.